src/components/common/Navbar/Navbar.tsx

```diff
@@ -42,30 +42,51 @@ function NavLinks({ onClickLink }: NavLinksProps) {
 
 function Navbar() {
   const [showMenu, setShowMenu] = useState(false)
```

Comment:
```suggestion
  const [showMenu, setShowMenu] = useState<boolean>(false)
```

Reply:
With the type inference, we don't need to use the generics in this case.

![Screen Shot 2022-02-21 at 10 00 00](https://user-images.githubusercontent.com/51174217/154960062-18f51475-f364-435b-ba36-8b65fe8a337b.png)